src/frontend/auth0_templates/invite_new_user_email_template_prod.html

```diff
@@ -17,23 +17,22 @@
         id="bodyTable"
       >
         <tr>
-          <td bgcolor="#000000">
-          	<div style='height:58px;'>
-              <a href="https://czgenepi.org/CZGenEpiLogoWhite.png">
-              <img
-                   src="https://czgenepi.org/CZGenEpiLogoWhite.png"
-                   alt="czgenepi-logo" 
-                   title="czgenepi-logo"
-                   width="150"
-                   style="display:block;float:left;margin:10px;border: 0;height: auto;outline: none;text-decoration: none;"
-               />
-              </a>
-	        </div>
-         	<div style='height:90px;border-bottom: 5px solid #511CC1;overflow:hidden;'>
-            	<div style='width:30%;height:90px; float:left;'></div>
-                <div style='background-image:url("https://czgenepi.org/phyloTreeBackground.png");background-color:black; height:90px;width:70%;float:right;background-position:right;background-size:cover;'>
-                </div> 
-        	</div>
+          <td>
+            <div style='height:90px;border-bottom:5px solid #511CC1;overflow:hidden;display:flex;'>
+              <div style='background-color:black;'>
```

Comment:
I was reading that some colors arn't recognized in emails unless you give hex value
```suggestion
              <div style='background-color:#000000;'>
```